lib/milestone-db/memory.js

```diff
@@ -22,12 +23,17 @@ function MemoryMilestoneDB(options) {
 MemoryMilestoneDB.prototype = Object.create(MilestoneDB.prototype);
 
 MemoryMilestoneDB.prototype.getMilestoneSnapshot = function (collection, id, version, callback) {
+  if (!callback) callback = function () {};
+  if (!collection) return process.nextTick(callback, new ShareDBError(4001, 'Invalid collection'));
+  if (!id) return process.nextTick(callback, new ShareDBError(4001, 'Invalid ID'));
+  if (!this._isValidVersion(version)) return process.nextTick(callback, new ShareDBError(4001, 'Invalid version'));
+
   var milestoneSnapshots = this._getMilestoneSnapshotsSync(collection, id);
 
   var milestoneSnapshot;
   for (var i = 0; i < milestoneSnapshots.length; i++) {
     var nextMilestoneSnapshot = milestoneSnapshots[i];
-    if (nextMilestoneSnapshot.v <= version || version === null) {
+    if (nextMilestoneSnapshot.v <= version || version == null) {
```

Comment:
Why change to `==` here?

Reply:
This was because of the fact that `undefined` was treated as a valid version, but now it's not, so I'll change this back.